src/plugins/elasticsearch/lib/create_proxy.js

```diff
@@ -16,10 +16,11 @@ function createProxy(server, method, route, config) {
     handler: {
       proxy: {
         mapUri: mapUri(server),
-        passThrough: true,
         agent: createAgent(server),
-        xforward: true,
```

Comment:
I agree with the sentiment, but in this case, `xforward` is already enabled on the proxy, so we're actually talking about *removing* that feature without any means for people to turn it back on.

It seems like a different issue than this PR is addressing, so why don't we keep it on in this PR and address it in a separate issue/PR?  It'd probably be something that we would at least want to make configurable since there is no workaround for it if we just outright remove it.